python/lsst/ip/isr/isrFunctions.py

```diff
@@ -216,6 +217,108 @@ def growMasks(mask, radius=0, maskNameList=['BAD'], maskValue="BAD"):
         fpSet.setMask(mask, maskValue)
 
 
+def maskITLEdgeBleed(ccdExposure, itlEdgeBleedSatMinArea,
```

Comment:
Looking at the tests below I think that everything except ccdExposure should be a keyword with a default.  It's a bit redundant with the task configs, but I think it'll be clearer here what the intended values are.

Reply:
done except for ccdExposure and badAmpDict